docs/tutorials/upgrading-ev3dev.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Upgrading Ev3dev
```

Comment:
again, I understand the reason of capitalization... 

Again the same feeling that it should be ev3dev